common-api/src/main/java/org/pmiops/workbench/cdr/model/DbConcept.java

```diff
@@ -49,6 +47,33 @@ public DbConcept(DbConcept a) {
         .synonymsStr(a.getSynonymsStr());
   }
 
+  // Used from JQL queries in DbConcept
```

Comment:
Eventually i think we will be replacing DbConcept with DbCriteria

Reply:
Good to know